blob/blob.go

```diff
@@ -591,6 +591,88 @@ func (b *Bucket) List(opts *ListOptions) *ListIterator {
 	return &ListIterator{b: b, opts: dopts}
 }
 
+// FirstPageToken is the pageToken to pass to ListPage to retrieve the first page of results.
+var FirstPageToken = []byte("first page")
+
+// ListPage returns a page of ListObject results for blobs in a bucket, in lexicographical
+// order of UTF-8 encoded keys.
+//
+// To fetch the first page, pass FirstPageToken as the pageToken. For subsequent pages, pass the pageToken
+// returned from a previous call to ListPage. It is not possible to "skip ahead" pages.
+//
+// Each call will return pageSize results, unless there are not enough blobs to fill the
+// page. If there are no more blobs available, the returned pageToken will be nil. Calling List
+// Page with a nil pageToken will return io.EOF.
```

Comment:
Would it be clearer to talk about an "empty pageToken", given that it's a slice type? nil is a bit confusing here

Reply:
Done.